config/locales/en.yml

```diff
@@ -166,7 +167,7 @@ en:
       translate_ex2: "Read our %{translation_instructions_link} on how to get started."
       translation_instructions_link_text: "translation instructions "
       voice_header: "Provide your voice"
-      voice_ex1: "Our project is run entirely by community volunteers. We use a tool called Loomio to gauge how our contributors feel about ideas and proposals for the project. "
+      voice_ex1: "Our project is run entirely by community volunteers. We use a tool called Discourse to engage in conversations with our contributors about ideas and proposals."
```

Comment:
I guess that the people who are interested by this will realize this very quickly. :)